apps/main-landing/src/app/creators/donate/schema.ts

```diff
@@ -13,7 +13,7 @@ export const createSendPayloadSchema = (allowedChainIds: number[]) => {
     amount: z
       .number()
       .gte(MIN_SEND_AMOUNT, `Value must be at least $${MIN_SEND_AMOUNT}`),
-    tokenAddress: hexSchema,
+    tokenAddress: z.string(),
```

Comment:
I think we should keep `hexShema` here, and make an update in the `donate/content.tsx` (https://github.com/idriss-xyz/core/pull/184/files#r1874046247)

Reply:
I created new schema for form and bring hex back